nodejs/week5/src/frontend/pages/home.js

```diff
@@ -0,0 +1,175 @@
+window.handleHomeRequest = () => {
+  document.body.innerHTML = `
+  <header>
+  <ul class="mealApp">
+    <a href="/" data-navigo>Home</a>
+    <a href="meals" data-navigo>Meals</a>
+    <a href="meal/1" data-navigo>Reservation</a>
+    <a href="review" data-navigo>Review</a>
+    </ul>
+
+    <div class="container">
+      <div class="row">
+        <h1>Welcome to CHARLIE'S RESTAURANT</h1>
+
+        <p>We showcase our best meals offers for our larger Copenhagen customers. 
+        Our services are very pocket friendly in terms of pricing, and we are within your reach, WELCOME!.</p>
+        </header>
+       
+       
+       
+    <div class="mealsGallery">
+<div class="galleries">
+<h4>OUR MEALS</h4>
+</div>
+<div class="mealOffers">
+
+<div class="mealRows">
+  <div class="col-1">
+  <a href="meals" data-navigo><img class ="meal1"src = "https://i.imgur.com/FFsAHlO.jpeg" alt="beef stew"></a>  
```

Comment:
All of these meals are hardcoded, is that correct? I imagine the ideal is that these are actual meals being pulled from the db, passed from the backend, and then displayed on the frontend...